content/docs/apps/static-egress.md

```diff
@@ -5,15 +5,22 @@ menu:
 title: Communicating with external services
 ---
 
-If your cloud.gov-hosted application needs access to an external service, that service must be accessible over the internet. All traffic from cloud.gov applications will transit the internet.
+You can set up your cloud.gov application to communicate with any external service that is accessible over the internet. For example, you can integrate your application with services in your agency infrastructure or with APIs from commercial providers.
+
+All traffic from cloud.gov applications will transit the internet. You can secure the communication with authentication and with encryption in transit, and you can limit access to specific IP ranges.
 
 ## Securing communications with external services
 
 You can set up your cloud.gov application to communicate with any external service available over the internet. To ensure only your application can communicate with the external service, the service should be configured to authenticate each client using client-specific credentials.
 
+Authentication options:
+
+* **cloud.gov client application connecting to external services:** If your application on cloud.gov needs to be a client of an existing server application outside of cloud.gov, your application on cloud.gov can have a TLS certificate that the server application validates, or you can use application-level authentication (such as Authorization headers). In addition, you can configure outside services to accept connections from the [cloud.gov egress address ranges](#cloud-gov-egress-ranges). There are no restrictions on outbound UDP or TCP connections, such as DNS (UDP), HTTP or HTTPS (TCP).
```

Comment:
Could these two paragraphs benefit from a simple diagram in addition to the text? Otherwise, it takes a few readings to grok (at least for me).

Reply:
A diagram sounds great to me! I don't know how to model this in a diagram though.